python/py-beancount/Portfile

```diff
@@ -24,7 +24,7 @@ homepage            http://furius.ca/beancount/
 master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
 distname            ${python.rootname}-${version}
```

Comment:
`master_sites` and `distname` can be removed, these are the defaults set by the `python` PG